### ISSUE TITLE
Add option to print test output to the console

### ISSUE THREAD

**Issue by kangasta** [UpCloudLtd/mdtest#14]
_Copied from [comment](https://github.com/UpCloudLtd/mdtest/issues/14#issuecomment-3957740779)_

There is an option to generate a XUnit output, so some logic towards this already exists and would be useful when implementing these

- Could be enabled with `--output-to-terminal` or `--output-to-console` flag.
- This could become messy if multiple tests are running in parallel, so maybe good to require that only one test is running concurrently. I.e., `--jobs` is set to one or user is only targeting single file.
- If output is printed live (i.e., when printed during sh script execution instead of after it has finished) the output will collide with the progress logging. In that case `DisableAnimations` option of progress can be used. See `upctl`logic when `--debug` is set for similar example: [UpCloudLtd / upcloud-cli : internal/config/progress.go](https://github.com/UpCloudLtd/upcloud-cli/blob/v3.30.0/internal/config/progress.go#L23)

_Files output split to #88_

**Comment by kangasta** [comment]
Hello! 👋 Yes, contributions welcome! The project is still in quite early phase so we do not have much documentation contributors at the moment.

_I'll add the thoughts to the issue description as well:_

There is an option to generate a XUnit output, so some logic towards this already exists and would be useful when implementing these. Would be maybe good to split this into two separate pull requests:

### Option to print output to the console

- Could be enabled with `--output-to-terminal` or `--output-to-console` flag.
- This could become messy if multiple tests are running in parallel, so maybe good to require that only one test is running concurrently. I.e., `--jobs` is set to one or user is only targeting single file.
- If output is printed live (i.e., when printed during sh script execution instead of after it has finished) the output will collide with the progress logging. In that case `DisableAnimations` option of progress can be used. See `upctl`logic when `--debug` is set for similar example: [UpCloudLtd / upcloud-cli : internal/config/progress.go](https://github.com/UpCloudLtd/upcloud-cli/blob/v3.30.0/internal/config/progress.go#L23)

### Option to print output to files

- Could be enabled with `--output-to-directory` flag. Then, if user specifies a directory, create `.out.txt` file with the test output for each `.md` file that is executed.
- Probably simpler (and good enough) if this is done after the test execution instead of streaming the output to the files.
